app/services/reject_application_by_default.rb

```diff
@@ -14,5 +14,6 @@ def call
 
     StateChangeNotifier.call(:reject_application_by_default, application_choice: application_choice)
     SendRejectByDefaultEmailToProvider.new(application_choice: application_choice).call
+    SendCandidateRejectionEmail.new(application_choice: application_choice).call
```

Comment:
I'm wondering how this will work in practice. If the candidate has 3 application choices, they will get 3 emails, but the last one will be different. The "application_rejected_awaiting_decisions" email is technically possible, but only by milliseconds.

Reply:
Yes the RBDs would probably all happen at once for candidates with more than one. I wondered whether we should refactor the `RejectApplicationsByDefault` and `RejectApplicationByDefault` to batch these together somehow but not sure how that would work - it would certainly need new content. Is that something for a follow-up or should we try to tackle it here?

We could defer the rejection emails by a few seconds to give time for `RejectApplicationsByDefault` to finish its cycle. That would at least mean that the multiple emails would be consistent because they would be rendered after all the application choices had been rejected.

Reply:
@tvararu & @tijmenb I've changed the way we call the candidate mailer from the `RejectApplicationByDefault` to put in a short delay (1 minute) so that we can avoid the problem that we discussed where all but the last of a set of multiple RBD emails are misleading because they imply that there are still pending decisions. I added an explicit order to the the query that fetches the application choices to process so that the application choices for a given candidate are not potentially spread out over a large result set.